buildtools/tinygo/wasi-libc.Dockerfile

```diff
@@ -5,8 +5,9 @@ FROM ghcr.io/corazawaf/coraza-proxy-wasm/buildtools-wasi-sdk:main
 
 RUN apt-get install -y git
 
-RUN git clone https://github.com/tinygo-org/tinygo --branch dev
+RUN git clone https://github.com/anuraaga/tinygo --branch custom-gc
```

Comment:
I see we do this very often. Maybe it is time to host the fork in coraza?

Reply:
Yeah we may need to, let's give it a bit more time though as it's a significant move